scripts/generate_landings_risk_report.py

```diff
@@ -846,7 +846,12 @@ def go(self, days: int) -> None:
 
         meta_bugs = self.get_meta_bugs(days)
 
-        last_modified = db.last_modified(bugzilla.BUGS_DB)
```

Comment:
Here we actually want to propagate the exception, as it should never happen here that the Last-Modified is not available.